card/spec/lib/card/query_spec.rb

```diff
@@ -397,8 +403,8 @@
 
   describe "relative" do
     it "should clean wql" do
-      wql = Card::Query.new( :part=>"_self",:context=>'A' )
-      expect(wql.query[:part]).to eq('A')
+      query = Card::Query.new( :part=>"_self",:context=>'A' )
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator '=>'.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.